src/bin/wc.rs

```diff
@@ -0,0 +1,149 @@
+use std::env;
+use std::fs::File;
+use std::io;
+use std::io::BufReader;
+use std::io::Read;
+
+struct Config {
+    bytes: bool,
+    lines: bool,
+    chars: bool,
+    words: bool,
+}
+
+struct Output {
+    bytes: u32,
+    lines: u32,
+    chars: u32,
+    words: u32,
+    filename: Option<String>,
+}
+
+impl Config {
+    pub fn new() -> Config {
+        let (bytes, lines, chars, words) = (false, false, false, false);
+        Config {
+            bytes,
+            lines,
+            chars,
+            words,
```

Comment:
style: i'd recommend building the struct by setting the fields directly like
```rust
Config {
    bytes: false,
    lines: false,
    ....
}
```

Reply:
Makes sense. Done.